DiscordChatExporter.Core.Rendering/Resources/HtmlCore.css

```diff
@@ -57,6 +57,70 @@ img {
     border-radius: 3px;
 }
 
+.spoiler.hidden {
```

Comment:
This stylesheet follows BEM notation, so please rename this to `.spoiler--hidden` since it's a modifier.